test/test_cli.rb

```diff
@@ -348,47 +348,6 @@ def test_download_zip
     test_download
   end
 
-  def test_new
```

Comment:
Hmm, not ideal to remove this spec completely. Can we do something simple here, like put a SUPER minimal scaffold in `SKELETON_DIR` (or a stubbed version of SKELETON_DIR) and then stub `online?` to return `false`? Then we can test without triggering a connection to GitHub